templates/crowdstrike-block.hbs

```diff
@@ -1,193 +1,334 @@
-      href='#'
-    >
-      Crowdstrike
-    </a>
-  </li>
-
-  {{#if block.userOptions.searchIoc}}
+{{#if details.errorMessage}}
+  <div class='mt-2'>
+    <div class='alert alert-danger'>
+      <pre>{{details.errorMessage}}</pre>
+    </div>
+    {{#if (not (eq details.allowRetry false))}}
+      <button type='button' class='p-btn btn btn-sm btn-secondary retry-btn mt-2' {{action 'retryLookup'}}>
+        {{#if running}}
+          {{fa-icon icon='spinner-third' fixedWidth=true spin=true}}
+        {{else}}
+          Retry Search
+        {{/if}}
+      </button>
+      <div class='mt-1'>
+        <span class='p-footnote'>
+          You can retry your search by pressing the "Retry Search" button.</span>
+      </div>
+    {{/if}}
+  </div>
+{{else}}
+  <ul class='nav nav-tabs'>
     <li class='nav-item'>
       <a
-        {{action 'changeTab' 'crowdstrikeIoc'}}
-        class='nav-link {{if (eq activeTab "crowdstrikeIoc") "active"}}'
+        {{action 'changeTab' 'crowdstrike'}}
+        class='nav-link {{if (or (eq activeTab "crowdstrike") (not activeTab)) "active"}}'
         href='#'
       >
-        Crowdstrike IOC
+        Crowdstrike
       </a>
     </li>
-  {{/if}}
-</ul>
 
-{{#if (eq activeTab 'crowdstrike')}}
-  {{#if details.detections}}
-    <div>
-      Displaying
-      {{details.detections.length}}
-      of
-      {{details.meta.totalResults}}
-      results.
-    </div>
-  {{else}}
-    <div class='m-1'>
-      No Results.
-    </div>
-  {{/if}}
-  {{#each details.detections as |detection detectionIndex|}}
-    {{#if (gt detectionIndex 0)}}
-      <hr />
+    {{#if (and block.userOptions.searchIoc details.devices)}}
+      <li class='nav-item'>
+        <a
+          {{action 'changeTab' 'crowdstrikeIoc'}}
+          class='nav-link {{if (eq activeTab "crowdstrikeIoc") "active"}}'
+          href='#'
+        >
+          Crowdstrike IOC
+        </a>
+      </li>
     {{/if}}
-    <div class='p-title'>
-      {{fa-icon 'search' fixedWidth=true}}
-      Detection&nbsp;
-      <a href='{{detection.__url}}' class='p-link detection-title'>View in Falcon
-        {{fa-icon 'external-link-square' fixedWidth=true class='external-link'}}</a>
-    </div>
-    {{#each detectionProperties as |prop|}}
-      <div>
-        <span class='p-key'>{{humanize prop}}:</span>
-        <span class='p-value'>{{get detection prop}}</span>
+    {{#if details.devices}}
+      <li class='nav-item'>
+        <a
+          {{action 'changeTab' 'hosts'}}
+          class='nav-link {{if (or (eq activeTab "hosts") (not activeTab)) "active"}}'
+          href='#'
+        >
+          Containment
+        </a>
+      </li>
+    {{/if}}
+  </ul>
+  {{#if (eq activeTab 'crowdstrike')}}
+    {{#if details.detections}}
+      {{#if (and details.detections.length details.meta.totalResults)}}
+        <div class="display-results">
+          Displaying
+          {{details.detections.length}}
+          of
+          {{details.meta.totalResults}}

```

Comment:
So I'm finding that `details.meta.totalResults` is always empty.  Any idea why 🤔?